subgraphs/compound-forks/cream/src/mapping.ts

```diff
@@ -0,0 +1,186 @@
+    );
+    _handleMarketListed(marketListedData, event);
+    return;
+  }
+
+  let underlyingTokenAddrResult = cTokenContract.try_underlying();
+  if (underlyingTokenAddrResult.reverted) {
+    log.warning(
+      "[handleMarketListed] could not fetch underlying token of cToken: {}",
+      [cTokenAddr.toHexString()]
+    );
+    return;
+  }
+  let underlyingTokenAddr = underlyingTokenAddrResult.value;
+  let underlyingTokenContract = ERC20.bind(underlyingTokenAddr);
+  _handleMarketListed(
+    new MarketListedData(
+      protocol,
+      new TokenData(
+        underlyingTokenAddr,
+        getOrElse<string>(cTokenContract.try_name(), "unknown"),
+        getOrElse<string>(cTokenContract.try_symbol(), "unknown"),
+        cTokenDecimals
+      ),
+      new TokenData(
+        cTokenAddr,
+        getOrElse<string>(underlyingTokenContract.try_name(), "unknown"),
+        getOrElse<string>(underlyingTokenContract.try_symbol(), "unknown"),
+        getOrElse<i32>(underlyingTokenContract.try_decimals(), 0)
+      ),
+      cTokenReserveFactorMantissa
+    ),
+    event
+  );
+}
+
+export function handleNewCollateralFactor(event: NewCollateralFactor): void {
+  _handleNewCollateralFactor(event);
+}
+
+export function handleNewLiquidationIncentive(
+  event: NewLiquidationIncentive
+): void {
+  let protocol = getOrCreateProtocol();
+  _handleNewLiquidationIncentive(protocol, event);
+}
+
+export function handleNewReserveFactor(event: NewReserveFactor): void {
+  _handleNewReserveFactor(event);
+}
+
+export function handleMint(event: Mint): void {
+  _handleMint(comptrollerAddr, event);
+}
+
+export function handleRedeem(event: Redeem): void {
+  _handleRedeem(comptrollerAddr, event);
+}
+
+export function handleBorrow(event: BorrowEvent): void {
+  _handleBorrow(comptrollerAddr, event);
+}
+
+export function handleRepayBorrow(event: RepayBorrow): void {
+  _handleRepayBorrow(comptrollerAddr, event);
+}
+
+export function handleLiquidateBorrow(event: LiquidateBorrow): void {
+  _handleLiquidateBorrow(comptrollerAddr, event);
+}
+
+export function handleAccrueInterest(event: AccrueInterest): void {
+  let marketAddress = event.address;
+  let cTokenContract = CToken.bind(marketAddress);
+  let protocol = getOrCreateProtocol();
+  let oracleContract = PriceOracle.bind(
+    Address.fromString(protocol._priceOracle)
+  );
+  let updateMarketData = new UpdateMarketData(
+    cTokenContract.try_totalSupply(),
+    cTokenContract.try_exchangeRateStored(),
+    cTokenContract.try_totalBorrows(),
+    cTokenContract.try_supplyRatePerBlock(),
+    cTokenContract.try_borrowRatePerBlock(),
+    oracleContract.try_getUnderlyingPrice(marketAddress),
+    unitPerYear
+  );
+  _handleAccrueInterest(updateMarketData, comptrollerAddr, event);
+}
+
+function getOrCreateProtocol(): LendingProtocol {
+  let comptroller = Comptroller.bind(comptrollerAddr);
+  let protocolData = new ProtocolData(
+    comptrollerAddr,
+    "Cream Finance",
```

Comment:
Can you change name to "CREAM Finance" and slug to "cream-finance"? We should also change the folder name to align with slug

Reply:
how about just "Cream" and "cream"?